### PR TITLE
Script for updating stops and transport tree in Strapi

### PR DESCRIPTION
**Pull request by hiba9201**
- Added script for updating stops and transport tree. In the future we're gonna write a scheduler to update them once per day
- Created `common` package with data that is shared between different packages. For example common types between server and client or server and schedulers
- Also moved base tsconfigs to `common`
- Refactored ekaterinburg rf model after moving transport tree to strapi
- Little fixes in client because of check types and problems with cms

### CODE REVIEW

File: common/strapi/create-methods.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+            if (body.error) {
+                throw new Error(JSON.stringify(body.error));
+            }
+        } catch (e) {
+            console.log(e);
+        }
+    }
+
+    async function getAll() {
```
Comment (a-chabin) [inline common/strapi/create-methods.ts:72]: Maybe we can add a function that receives data by key?

https://github.com/ekaterinburgdev/transport/pull/31#discussion_r1146363248

File: common/strapi/create-methods.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        try {
+            const resp = await fetch(
+                `${strapiUrl}/api/${contentType}s?pagination[pageSize]=2000`,
```
Comment (a-chabin) [inline common/strapi/create-methods.ts:75]: Let's add a variable `const REQUEST_PAGINATION_SIZE = 2000`

File: common/strapi/create-methods.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import fetch from 'node-fetch';
+
+import { StrapiContentTypes } from '../types/strapi';
+import { strapiUrl } from './constants';
+
+export function createStrapiMethods(contentType: StrapiContentTypes, jwt?: string) {
```
Comment (a-chabin) [inline common/strapi/create-methods.ts:6]: 👍

File: common/strapi/create-methods.ts (hunk 1 above)
Comment (hiba9201) [reply common/strapi/create-methods.ts:72]: Why? We don't need right now. At the moment we only have data that just shows where stops are. And to show all stops on map we need to request them all

Comment (hiba9201) [reply common/strapi/create-methods.ts:72]: I'm going to add it when we create table with data about transport. With photos and this additional info
image

Comment (hiba9201) [reply common/strapi/create-methods.ts:72]: Or you want to use it for articles?

**Pull request merged**